orchestrator-salt/src/main/resources/salt/salt/datalake_resize/scripts/check_atlas_updated.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+LOGFILE=/var/log/dl_resize_check_atlas_updated.log
```

Comment:
if this is a necessary log for engesc investigation then please add it the log bundle collection. or at least ensure the output of the script is also passed to standard output or error

Reply:
I will look into this.